cmd/root.go

```diff
@@ -49,14 +49,14 @@ var (
 
 // RootCmd root level flags and commands
 var RootCmd = &cobra.Command{
-	Use:   "kompose",
-	Short: "A tool helping Docker Compose users move to Kubernetes",
-	Long:  `Kompose is a tool to help users who are familiar with docker-compose move to Kubernetes.`,
+	Use:           "kompose",
+	Short:         "A tool helping Docker Compose users move to Kubernetes",
+	Long:          `Kompose is a tool to help users who are familiar with docker-compose move to Kubernetes.`,
+	SilenceErrors: true,
```

Comment:
I would like to avoid duplicated outputs by enabling `SilenceErrors`.
Does this make sense?

Ref: there are two places to be used:
- https://github.com/spf13/cobra/blob/86f8bfd7fef868a174e1b606783bd7f5c82ddf8f/command.go#L940
- https://github.com/spf13/cobra/blob/86f8bfd7fef868a174e1b606783bd7f5c82ddf8f/command.go#L969